modules/nf-core/fastqc/main.nf

```diff
@@ -22,8 +22,8 @@ process FASTQC {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    printf "%s\\n" $reads | while read f; do ln -s \$f ${prefix}_\$(basename \$f) ; done
-    fastqc $args --threads $task.cpus ${prefix}_*
+    printf "%s\\n" $reads | while read f; do [[ \$(basename \$f) =~ ^${prefix}.* ]] && ln -s \$f \$(basename \$f) || ln -s \$f ${prefix}_\$(basename \$f) ; done
```

Comment:
```bash
&& ln -s \$f \$(basename \$f)
```
Why is this here? It's reversing the `stageAs`.

Reply:
See also your [other comment](https://github.com/nf-core/modules/pull/2465#issuecomment-1305251788):

> There's an odd `1/` and `2/` in the name.

This is where that numeric prefix directory is coming from.

Reply:
Ah, I see. That's a good point about not including original files, but if we remove the `stageAs` then we can simplify this expression, and the `fastqc ${prefix}*` ( already there ) should exclude any files we don't want to run on. The `stageAs` is normally used to prevent filename collisions, so if we leave it in, we need to make sure the symlinking doesn't also reintroduce this issue.